README.md

```diff
@@ -0,0 +1,182 @@
+# React Lazylog
+
+React component that loads and views remote text in the browser lazily and efficiently.  
+Forked from
+[mozilla-frontend-infra/react-lazylog](https://github.com/mozilla-frontend-infra/react-lazylog).
+This fork is maintained by a team at [Spire Global, Inc.](https://github.com/nsat) for internal usage. Outside contributions are
+**not** currently being reviewed or merged.
+
+## Features
+
+-   Efficient scrolling performance thanks to [react-virtualized](https://github.com/bvaughn/react-virtualized)
+-   Able to load large files upwards of 100MB without crashing the browser
+-   Parses, colorizes, and styles ANSI escapes within content
+-   Supports remote text files as well as chunked/streamed responses
+-   Line highlighting
+-   Customizable styling
+-   Searching through log
+-   Works in latest browser versions, including iOS Safari and Android Chrome
+-   v3+ requires React 16.3+
+
+## Getting started
+
+You can install `react-lazylog` via Yarn or npm:
+
+```bash
+# If using Yarn:
+yarn add react-lazylog
```

Comment:
This more has to do with default instructions on how to install it into another repo, so other projects using yarn is fine. But, it actually isn't quite true anymore since `react-lazylog` doesn't point to our package anymore. So, I'll update that.